lib/plugins/extension/style.less

```diff
@@ -378,6 +378,14 @@
     text-align: center;
     display: block;
     width: 60%;
+
+    label.block {
+        margin-bottom: .8em;
+    }
+
+    input {
+        width: 25em;
```

Comment:
Can we avoid this fixed width? Is it necessary? Could it be expressed as a percentage instead?

Reply:
After comparing two form classes, I found that `Doku_Form::form_textfield()` renders `<br>`, but `Form\Form::addTextInput()` does not. Now, `AddTag('br')` makes additional css unnecessaty.